appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -142,28 +154,28 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
-	def 'Retrieve activity report of previous week'()
+	def 'Retrieve activity report of previous weeks'()
```

Comment:
I'm going to revert this change and instead make a test "Richard gets proper navigation links between day and week details". That'll be easier to understand.

Reply:
Done.